driver/web/authorization_admin_policy.csv

```diff
@@ -49,4 +49,8 @@ p, update_account-authorization, /core/admin/application/accounts/*/roles, (PUT)
 p, update_account-authorization, /core/admin/application/groups/*/accounts, (PUT)|(DELETE),
 p, update_account-authorization, /core/admin/application/permissions, (GET),
 p, update_account-authorization, /core/admin/application/roles, (GET),
-p, update_account-authorization, /core/admin/application/groups, (GET),
\ No newline at end of file
+p, update_account-authorization, /core/admin/application/groups, (GET),
+
+p, all_account-system-configs, /core/admin/application/accounts/*/system-configs, (GET)|(POST)|(PUT)|(DELETE), All account system config actions
+p, get_account-system-configs, /core/admin/application/accounts/*/system-configs, (GET), Get account system configs
+p, update_account-system-configs, /core/admin/application/accounts/*/system-configs, (PUT), Get account system configs
```

Comment:
Thanks for catching the comment, I will fix this

For the GET point, we actually aren't necessarily allowing everyone who can update to see the full contents since various parties may use these configs for different purposes and should not necessarily know what others have done with them.